icu4j/main/classes/core/src/com/ibm/icu/message2/Mf2Serializer.java

```diff
@@ -117,52 +117,27 @@ public String toString() {
 
     Mf2Serializer() {}
 
-    /**
```

Comment:
@internal/@deprecated tags are not required in package private classes. I deleted these unnecessary tags.